nomad/repo.py

```diff
@@ -94,18 +98,15 @@ def available(self):
 
     @cachedproperty
     def appliednames(self):
-       return [x for (x, ) in
-               self.engine.query('SELECT name FROM %s ORDER BY date' %
-                                 self.conf['nomad']['table'])]
+        q = 'SELECT name FROM {0} ORDER BY date'.format(
```

Comment:
I'm not going to get into a discussion on coding standards. If you only want the "old-style" (read: old) then I'll change them!